### ISSUE TITLE
Suggestion for GU audio: example for simple beep or implement bell via print \a

### ISSUE THREAD

**Issue by xciv** [pimoroni/pimoroni-pico#669]
I am wanting to implement a simple beep through the Galactic Unicorn audio but the audio subsystem is very complex to understand for this kind of basic requirement so an example in the documentation or some simpler code in the examples directory would be really useful.

Even better would be implementation of the print \a bell character!

**Comment by Gadgetoid** [comment]
You're right, it's awfully complex and could probably stand a channel reserved for some basic utility effects. Though hooking `print()` in MicroPython would probably be quite tricky.

In the mean time here's how to configure a channel to use as a basic beeper in MicroPython:

```python3
import time
from cosmic import CosmicUnicorn

cu = CosmicUnicorn()

# New beeper on channel 1
beeper = cu.synth_channel(1)

# Configure the beeper waveform and envelope
beeper.configure(
    waveforms=beeper.SINE + beeper.SQUARE,
    attack=0.05,
    decay=0.5,
    sustain=0,
    release=0,
    volume=1.0
)

# Configure the beeper frequency
beeper.frequency(440)

# Start the synth engine running
cu.play_synth()

# Beep lots
while True:
    beeper.trigger_attack()
    time.sleep(1.0)
```

Once configured you can call `trigger_attack()` where you need in order to make it beep.

`SQUARE + SINE` is mellow-ish, a pure `SQUARE` will be a little more shouty.
